src/index.js

```diff
@@ -88,9 +75,11 @@ const getFirstReplacementValueForNode = (obj, comparator, nodePath) => {
  * @return {undefined}
  */
 const processNode = (replacements, nodePath, replaceFn, comparator) => { // eslint-disable-line
-  const replacement = getFirstReplacementValueForNode(replacements, comparator, nodePath);
-  if (replacement) {
-    replaceAndEvaluateNode(replaceFn, nodePath, replacement);
+  const replacementKey = getSortedObjectPaths(replacements)
```

Comment:
Okay, I just added documentation.  However, I also changed the code to use `.find()` instead of `.filter().shift()`, so that it doesn't unnecessarily create an intermediate array.